docs/src/tutorial.md

```diff
@@ -88,20 +110,49 @@ renders delightful figures. Adding other geometries produces layers, which may
 or may not result in a coherent plot.
 
 ```@example 1
-plot(iris, x=:SepalLength, y=:SepalWidth,
-         Geom.point, Geom.line)
+plot(iris, x=:SepalLength, y=:SepalWidth, Geom.point, Geom.line)
 ```
 
 This is the grammar of graphics equivalent of "colorless green ideas sleep
 furiously". It is valid grammar, but not particularly meaningful.
 
+## Arrays
+
+If by chance your data are stored in Arrays instead of a DataFrame, fear not,
+identical plots can be created using an alternate `plot` signature:
+
+```julia
+plot(elements::Element...; mapping...)
+```
+
+Here, the keyword arguments directly supply the data to be ploted,
+instead of using them to indicate which columns of a DataFrame to use.
+
+```@example 1
+SepalLength = iris[:SepalLength]
+SepalWidth = iris[:SepalWidth]
+plot(x=SepalLength, y=SepalWidth, Geom.point,
+     Guide.xlabel("SepalLength"), Guide.ylabel("SepalWidth"))
+nothing # hide
+```
+
+Note that with the Array interface, extra elements must be included to specify the
+axis labels, whereas with a DataFrame they default to the column names.
```

Comment:
> Note that with the Array interface, extra elements must be included to specify the axis labels, whereas with a DataFrame they default to the column names.

What about:

Note that with the Array interface, the axis labels cannot be directly inferred so include explicit calls to `Guide.xlabel` etc to specify the axis names.